### PR TITLE
Remove microservice sdk c#

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

**Comment by BeateRixen** [comment]
@sharath-jk @kailol-sag please ignore

File: content/concepts/applications-bundle/microservices.md [hunk 1]
```diff
@@ -19,8 +19,4 @@ The hosting of the microservice is provided by {{< product-c8y-iot >}}. This way
...
 
-For detailed information on developing and deploying microservices on top of {{< product-c8y-iot >}} refer to the [Microservice SDK guide](/microservice-sdk/introduction) which provides information on the general concept of microservices in {{< product-c8y-iot >}} as well as specific guidance and examples for various programming languages.
-
-{{< c8y-admon-info >}}
-For Microservice developers {{< company-c8y >}} provides Microservice SDK in [Java programming language](/microservice-sdk/java) and [C# programming language](/microservice-sdk/cs) for .Net Core. Refer to the relevant sections in the [Microservice SDK guide](/microservice-sdk/introduction).
-{{< /c8y-admon-info >}}
+For detailed information on developing and deploying microservices on top of {{< product-c8y-iot >}} refer to the [Microservice SDK guide](/microservice-sdk/introduction) which provides information on the general concept of microservices in {{< product-c8y-iot >}} as well as specific guidance and examples for various programming languages. For Microservice developers {{< company-c8y >}} provides a Microservice SDK in the [Java programming language](/microservice-sdk/java) for .Net Core.
```
Comment (BeateRixen) [inline content/concepts/applications-bundle/microservices.md:22]:
```suggestion
For detailed information on developing and deploying microservices on top of {{< product-c8y-iot >}} refer to the [Microservice SDK guide](/microservice-sdk/introduction) which provides information on the general concept of microservices in {{< product-c8y-iot >}} as well as specific guidance and examples for various programming languages. For Microservice developers {{< company-c8y >}} provides a Microservice SDK in the [Java programming language](/microservice-sdk/java).
```

File: content/concepts/applications-bundle/microservices.md [hunk 2]
```diff
@@ -19,8 +19,4 @@ The hosting of the microservice is provided by {{< product-c8y-iot >}}. This way
...
 
-For detailed information on developing and deploying microservices on top of {{< product-c8y-iot >}} refer to the [Microservice SDK guide](/microservice-sdk/introduction) which provides information on the general concept of microservices in {{< product-c8y-iot >}} as well as specific guidance and examples for various programming languages.
-
-{{< c8y-admon-info >}}
-For Microservice developers {{< company-c8y >}} provides Microservice SDK in [Java programming language](/microservice-sdk/java) and [C# programming language](/microservice-sdk/cs) for .Net Core. Refer to the relevant sections in the [Microservice SDK guide](/microservice-sdk/introduction).
-{{< /c8y-admon-info >}}
+For detailed information on developing and deploying microservices on top of {{< product-c8y-iot >}} refer to the [Microservice SDK guide](/microservice-sdk/introduction) which provides information on the general concept of microservices in {{< product-c8y-iot >}} as well as specific guidance and examples for various programming languages. For Microservice developers {{< company-c8y >}} provides a Microservice SDK in the [Java programming language](/microservice-sdk/java).
```
Comment (rssag) [inline content/concepts/applications-bundle/microservices.md:22]:
```suggestion
For detailed information on developing and deploying microservices on top of {{< product-c8y-iot >}} refer to the [Microservice SDK guide](/microservice-sdk/introduction) which provides information on the general concept of microservices in {{< product-c8y-iot >}} as well as specific guidance and examples for various programming languages. For microservice developers {{< company-c8y >}} provides a Microservice SDK in the [Java programming language](/microservice-sdk/java).
```

**Review by rssag — APPROVED** [review]
1 suggestion + approval

**Review by MWindel — APPROVED** [review]

**Pull request merged**